pandas/tests/io/excel/test_readers.py

```diff
@@ -62,14 +79,16 @@ def test_usecols_int(self, read_ext, df_ref):
 
         # usecols as int
         with tm.assert_produces_warning(FutureWarning,
-                                        check_stacklevel=False):
+                                        check_stacklevel=False,
+                                        raise_on_extra_warnings=False):
```

Comment:
why does this have an extra warning?

Reply:
The installation of defusedxml (required by odfpy) unfortunately triggers `DeprecationWarning` and/or `PendingDeprecationWarning` during the XML parsing of ODS and Excel files. This is accounted for at a higher level during the parametrizataion of `read_engine` but needs to be allowed here as well or else this throws an error